### ISSUE TITLE
Er ontrbreken verschillende vertalingen.

### ISSUE THREAD

**Issue by Gaming-pandatje** [janfokke/StardewValleyDutch#32]
De vertaling van tools en artisan goods ontbreken.

![image](https://github.com/user-attachments/assets/ffd30063-7bda-4eac-a8a9-171ef4a4a511)
![image](https://github.com/user-attachments/assets/51df8b90-b382-4dd1-8841-fdd02ebab842)
![image](https://github.com/user-attachments/assets/70edc449-56a6-40be-be75-eadb7330aaea)

**Comment by azelion** [comment]
Yeah, hier was ik al bang voor. In het nieuwe jaar ga ik er naar kijken!

**Comment by azelion** [comment]
De vertalingen zouden toegevoegd moeten zijn https://github.com/janfokke/StardewValleyDutch/tree/v1_6_18

**Comment by Gaming-pandatje** [comment]
Ik snap niet helemaal hoe ik dit moet downloaden zou je het kunnen toevoegen als een nieuwe (alpha) release zodat ik het kan downloaden als en zip bestand. Nog eens bedankt voor de tijd die je hier in steekt.

**Comment by azelion** [comment]
https://github.com/janfokke/StardewValleyDutch/releases/tag/V1.6.18
?

**Comment by azelion** [comment]
https://github.com/janfokke/StardewValleyDutch/releases/tag/V1.6.18
Zou je kunnen checken of 't gefixed is?
